testing/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+cmake_minimum_required (VERSION 2.8)
+
+# projectname is the same as the main-executable
+project(testing)
+find_package(GTest)
+
+If(${GTest_FOUND})
+  message(STATUS Found GTest)
+else()
+  Build_GTEST()
```

Comment:
I think you have to add a "include(test_config)" in order for Build_GTEST() to be visible.